assets/_sass/components/_about-table-contents.scss

```diff
@@ -0,0 +1,36 @@
+.about-table-contents {
+  & i {
+    color: $color-dust;
+    font-weight: bold;
+    position: relative;
+    top: 2px;
+    left: -2px;
+  }
+
+  ul {
+    li {
+      list-style-type: none;
+      line-height: 2;
+      &:first-of-type {
+        @extend %btn;
+        text-transform: uppercase;
+      }
+      & i {
+        @include font-size(20px);
+      }
+      &:nth-of-type(n + 2) a:hover {
+        color: $color-dust;
+      }
+    }
+    padding-inline-start: calculate-rem(24px);
```

Comment:
- This should be `padding-left` to ensure cross-browser compatibility.
- Any styles being applied to the parent selector (the `ul`) should be listed first. So the order of this should go:
  1. ul styles
  2. ul:before styles
  3. li styles